gnoi/os/server_test.go

```diff
@@ -0,0 +1,47 @@
+package os
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/google/gnxi/gnoi/os/pb"
+)
+
+var (
+	server = initializeServer()
+)
+
+func initializeServer() *Server {
+	srv := NewServer("1")
+	srv.manager.Install("1.0.0a")
+	return srv
+}
+func TestActivate(t *testing.T) {
+	tests := []struct {
+		request *pb.ActivateRequest
+		want    *pb.ActivateResponse
+	}{
+		{
+			request: &pb.ActivateRequest{
+				Version: "1.0.0a",
+			},
+			want: &pb.ActivateResponse{Response: &pb.ActivateResponse_ActivateOk{}},
+		},
+		{
+			request: &pb.ActivateRequest{
+				Version: "99.0a",
+			},
+			want: &pb.ActivateResponse{Response: &pb.ActivateResponse_ActivateError{
+				ActivateError: &pb.ActivateError{Type: 1},
+			}},
+		},
+	}
+	for _, test := range tests {
+		got, _ := server.Activate(context.Background(), test.request)
+		diff := reflect.DeepEqual(test.want, got)
+		if !diff {
```

Comment:
use either cmp or pretty/messagediff, more context: refelct.DeepEqual it is sesntitive to changes in unexported fields.

Reply:
Thanks! Will change it